exchange/auction_test.go

```diff
@@ -192,6 +249,12 @@ type mockCache struct {
 	items []prebid_cache_client.Cacheable
 }
 
+type cacheComparator struct {
+	freq         int
+	expectedKeys []string
+	actualKeys   []string
+}
```

Comment:
I think cacheComparator struct is no longer used and should be removed.

Reply:
removed